doc/dev/niresframes.rst

```diff
@@ -36,27 +37,30 @@ required for the frame-typing and their associated keyword in the
 ``fitstbl`` key     Header Key
 ===============     ============
 ``idname``          ``OBSTYPE``
+``exptime``          ``ITIME``
 ===============     ============
 
 The criteria used to select each frame type are as follows:
 
-=============   ==========================================
-Frame           ``OBSTYPE``                               
-=============   ==========================================
-``science``     ``'Object'``, ``'object'``                             
-``dark``        ``'dark'``                                
-``pixelflat``   ``'domeflat'``                             
-``trace``       ``'domeflat'``                             
-``arc``         ``'Object'``, ``'object'``                             
-``tilt``        ``'Object'``, ``'object'``                             
-=============   ==========================================
+================   ==========================================================   =========
+Frame              ``OBSTYPE``                                                  ``ITIME``
+================   ==========================================================   =========
+``science``        ``'Object'``, ``'object'``                                   ``>60s``
+``standard``       ``'Object'``, ``'object'``, ``'standard'``, ``'telluric'``   ``<60s``
+``lampoffflats``   ``'dark'``, ``'Dark'``                                       Not used
+``pixelflat``      ``'domeflat'``                                               Not used
+``trace``          ``'domeflat'``                                               Not used
+``arc``            ``'Object'``, ``'object'``                                   Not used
+``tilt``           ``'Object'``, ``'object'``                                   Not used
+================   ==========================================================   =========
 
 .. note::
 
-    - Frames with type ``bias``, ``illumflat``, and ``pinhole`` are *not* typed.
+    - Frames with type ``bias``, ``dark``, ``illumflat``, and ``pinhole`` are *not* typed.
```

Comment:
Not for this PR, but do people typically take darks with NIRES data?  If so, is there a way to automatically identify them?

Reply:
@jhennawi -- what's the answer?

Reply:
Issue #1509 added.